lightly/utils/__init__.py

```diff
@@ -13,3 +13,5 @@
 from lightly.utils.io import load_embeddings
 from lightly.utils.io import load_embeddings_as_dict
 from lightly.utils.embeddings_2d import fit_pca
+from lightly.utils.benchmarking import BenchmarkModule
+from lightly.utils.benchmarking import knn_predict
```

Comment:
This is not necessary to do the benchmarks, right? It's already included in the `BenchmarkModule`. You just put it here if someone wants to use it on his own?

Reply:
yes. In general, I don't like that we hide many useful methods within lightly. Having as much as possible private does not help anyone using lightly to build a custom model.